### ISSUE TITLE
Icons not rendered with cdn embedded code

### ISSUE THREAD

**Issue by iwgx** [FortAwesome/Font-Awesome#11146]
When I'm using downloaded file (not cdn) it's work well.
But when I'm replaced it with cdn embedded code it's disappear.

So I thought the mistake is not from my code.

```
.......
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Dummy Website</title>
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link rel="stylesheet" href="resources/font/font-awesome-4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="resources/css/style.css">
</head>
.......
```
it's work and the icons rendered.

replaced with this
```
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Dummy Website</title>
    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <script src="https://use.fontawesome.com/{blablablabla}.js"></script>
    <link rel="stylesheet" href="resources/css/style.css">
</head>
```
now, it's not worked anymore

**Comment by tagliala** [comment]
you mean that all icons diseappear? Could you please check your development console and look for errors?

**Comment by IlhamWahabiGX** [comment]
oops, you are right. I got error on the console `GET https://use.fontawesome.com/de6c1fbc96.js net::ERR_NAME_NOT_RESOLVED`

I'm sorry but how to fix this?

**Comment by tagliala** [comment]
Where are you from?
Which internet provider are you using?

**Comment by IlhamWahabiGX** [comment]
From indonesia, pal. So this is internet connection problem? (but i thought my connection is fast enough) Or my country blocked it?

It's make sense because sometimes I must open a few of fontawesome page on Opera Browser ( with VPN ) from Chrome

**Comment by tagliala** [comment]
I think this could be related to your Country or your DNS Server.

I'm going to assign this issue to @robmadole, maybe there are some DNS issues or problems with the domain name registrar.

Could you please:
1. Check another connection
2. Post the output of `nslookup use.fontawesome.com`
3. Try the [Google Public DNS](https://developers.google.com/speed/public-dns/)

Sorry but I don't know how to provide help, it looks like a network issue

**Comment by robmadole** [comment]
@IlhamWahabiGX yeah just let us know what you find out from those questions @tagliala asked. (try all 4 if you can).

I did a quick check of our DNS systems and everything looks OK from our end but that could just mean the problem is deeper under the surface. Let's see what we can figure out.

**Comment by robmadole** [comment]
Closing this, @IlhamWahabiGX you can comment and we'll re-open if we need to.

**Comment by manh21** [comment]
Hi, I have the same issue, I use Cloudflare DNS (1.1.1.1) and still doesn't work at all.

![cdn prob](https://user-images.githubusercontent.com/31276236/40247723-8a409162-5af7-11e8-9e5d-22ea000daca7.JPG)

**Comment by manh21** [comment]
I am from Indonesia and my ISP is Telkom Indonesia

**Comment by tagliala** [comment]
@robmadole are you able to check the CDN status in Indonesia?

**Comment by robmadole** [comment]
Unfortunately, no, we can't help out too much with this one.

**Comment by manh21** [comment]
OK, I have found a solution

1. I do DNS Lookup for `use.fontawesome.com` and  I found `fontawesome-cdn.fonticons.netdna-cdn.com`
2. Instead of using `use.fontawesome.com` I use `fontawesome-cdn.fonticons.netdna-cdn.com`

> Not Working
> `<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">`

> Working
> <link rel="stylesheet" href="http://fontawesome-cdn.fonticons.netdna-cdn.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">

**Comment by tagliala** [comment]
@manh21 could you please check again tomorrow with `use.fontawesome.com`? maybe this is a temporary issue

**Comment by manh21** [comment]
sure, I will let you know

**Comment by manh21** [comment]
![nskoo](https://user-images.githubusercontent.com/31276236/40272918-a9f3b856-5be0-11e8-8863-0e543aafa7c7.JPG)
`use.fontawesome.com` doesn't work

**Comment by robmadole** [comment]
@manh21 that's a timeout issue, not a "server can't find". You aren't reaching that DNS server. Sorry, but this is a local or ISP issue that we can't help you with.

**Comment by morpig** [comment]
Hi, I'm bumping this as it's still happening right now from the same ISP as mentioned above. Is there any alternate domains/CDNs that we can use? fontawesome-cdn.fonticons.netdna-cdn.com works without any issues, but loading w/ SSL doesn't work.

**Comment by manh21** [comment]
@morpig Add this in hosts file
`23.111.9.35 use.fontawesome.com
50.31.246.1 fontawesome.com pro.fontawesome.com
94.31.29.154 use.fortawesome.com`

Directory C:\Windows\System32\drivers\etc

**Comment by morpig** [comment]
@manh21 Unfortunately that is not a good solution for all clients, as the user has to manually set the DNS from their OS.

**Comment by MeyaHoule** [comment]
file://sascgysrv02/UserData/participant/Desktop/Picture%20006.jpg

**Comment by ejhalpin** [comment]
I'm getting the same issue. I'm on the East Coast in the US. I just killed my kit and made a new one and that kit isn't working either. same issue.
GET https://kit.fontawesome.com/{kitcode}.js net::ERR_NAME_NOT_RESOLVED

**Comment by alekseyg** [comment]
Just started encountering this issue as well. I'm in Los Angeles, CA.

**Comment by robmadole** [comment]
We had a major DNS outage. We have switched providers but propagation is going to take time. Please refer to status.fortawesome.com for more info.

**Comment by tagliala** [comment]
Reopening for tracking purposes

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
